### DISCUSSION TITLE
warn Attempted import error: './Layout.module.less' does not contain a default export (imported as 'styles').

### DISCUSSION THREAD

**Discussion by talentAN** [gatsbyjs/gatsby#30333] [Help]
I've config gatsby-plugin-less correctly,  here's the code.
`
import React from 'react';
import styles from './Layout.module.less';

const Layout = ({ children }) => {
  console.info(styles);
  return <div className={styles.layout}>{children}</div>;
};

export default Layout;
`

**Comment by amantiwari1** [comment]
try this
```js
import React from 'react';
import styles from './Layout.module.less';

const Layout = ({ children }) => {
console.info(styles);
return ( <> {children} </> )

};
export default Layout;
```

and 
 you could add `default export  styles` in the `./Layout.module.less`

if not works
   send code in `./Layout.module.less`

**Comment by LekoArts** [comment]
Please see https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v2-to-v3/#css-modules-are-imported-as-es-modules
